src/UCommerce.SiteCore.Installer/Steps/MoveSitecoreConfigIncudes.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using System.Collections.Specialized;
+using Sitecore.Install.Framework;
+
+namespace Ucommerce.Sitecore.Installer.Steps
+{
+    public class MoveSitecoreConfigIncudes : IPostStep
+    {
+        public void Run(ITaskOutput output, NameValueCollection metaData)
+        {
+            var steps = new List<IPostStep>();
+
+            steps.Add(new MoveFileWeb(
+                "~/sitecore modules/Shell/ucommerce/install/configInclude/Sitecore.uCommerce.Databases.config",
+                "~/App_Config/include/Sitecore.uCommerce.Databases.config",
+                backupTarget: true));
+
+            steps.Add(new MoveFileWeb(
+                "~/sitecore modules/Shell/ucommerce/install/configInclude/Sitecore.uCommerce.Dataproviders.config",
+                "~/App_Config/include/Sitecore.uCommerce.Dataproviders.config",
+                backupTarget: true));
+
+            steps.Add(new MoveFileWeb(
+                "~/sitecore modules/Shell/ucommerce/install/configInclude/Sitecore.uCommerce.initialize.config",
+                "~/App_Config/include/Sitecore.uCommerce.initialize.config",
+                backupTarget: true));
+
+            steps.Add(new MoveFileWeb(
+                "~/sitecore modules/Shell/ucommerce/install/configInclude/Sitecore.uCommerce.Pipelines.HttpRequestBegin.config",
+                "~/App_Config/include/Sitecore.uCommerce.Pipelines.HttpRequestBegin.config",
+                backupTarget: true));
```

Comment:
Instead of calling the `Add` method for each step, could these just be included in the `List` constructor, thereby simplifying the code slightly?